src/lib.rs

```diff
@@ -13,3 +13,11 @@ pub mod metrics;
 #[cfg_attr(iroh_docsrs, doc(cfg(feature = "net")))]
 pub mod net;
 pub mod proto;
+
+#[cfg(feature = "rpc")]
+pub mod rpc;
+#[cfg(feature = "rpc")]
+pub use rpc::{
+    client::Client,
```

Comment:
why reexport, seems just the `rpc` module at the top would be cleaner?

Reply:
The reexport is just so that usage from iroh or other crates does not need the very long path.

instead of 

iroh_gossip::rpc::proto::Request

you can just do

iroh_gossip::RpcRequest

etc. in the main request and response enums.

Reply:
that's fair, but then please rename it to `RpcClient`